cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1519,335 +1519,372 @@ function Start_Containerd {
-# support kubernetes metadata properly. The current version does not recognizes
-# the local resource key "logging.googleapis.com/local_resource_id", and fails
-# to label namespace, pod and container names on the logs.
+# Installs the logging agent according to
 function Install-LoggingAgent {
-  # Remove the existing storage.json file if it exists. This is a workaround
-  # for the bug where the logging agent cannot start up if the file is
-  # corrupted.
-  Remove-Item `
-      -Force `
-      -ErrorAction Ignore `
-      ("$STACKDRIVER_ROOT\LoggingAgent\Main\pos\winevtlog.pos\worker0\" +
-       "storage.json")
-
   if (IsLoggingAgentInstalled) {
-    # Note: we should reinstall the Stackdriver agent if $REDO_STEPS is true
+    # Note: we should reinstall the agent if $REDO_STEPS is true
     # here, but we don't know how to run the installer without it prompting
-    # when Stackdriver is already installed. We dumped the strings in the
+    # when logging agent is already installed. We dumped the strings in the
     # installer binary and searched for flags to do this but found nothing. Oh
     # well.
-    Log-Output ("Skip: Stackdriver logging agent is already installed")
+    Log-Output ("Skip: Fluentbit logging agent is already installed")
     return
   }
   
-  # After a crash, the StackdriverLogging service could be missing, but its files will still be present
+  # After a crash, the $LOGGINGAGENT_SERVICE service could be missing, but its files will still be present
   Cleanup-LoggingAgent
 
-  $url = ("https://storage.googleapis.com/gke-release/winnode/stackdriver/" +
-          "StackdriverLogging-${STACKDRIVER_VERSION}.exe")
-  $tmp_dir = 'C:\stackdriver_tmp'
+
+  $url = ("https://fluentbit.io/releases/1.5/td-agent-bit-1.5.1-win64.zip")
+  $tmp_dir = 'C:\fluentbit_tmp'
   New-Item $tmp_dir -ItemType 'directory' -Force | Out-Null
-  $installer_file = "${tmp_dir}\StackdriverLogging-${STACKDRIVER_VERSION}.exe"
-  MustDownload-File -OutFile $installer_file -URLs $url
 
-  # Start the installer silently. This automatically starts the
-  # "StackdriverLogging" service.
-  Log-Output 'Invoking Stackdriver installer'
-  Start-Process $installer_file -ArgumentList "/S" -Wait
+  MustDownload-File -OutFile $tmp_dir\td.zip -URLs $url
 
-  # Install the record-reformer plugin.
-  Start-Process "$STACKDRIVER_ROOT\LoggingAgent\Main\bin\fluent-gem" `
-      -ArgumentList "install","fluent-plugin-record-reformer" `
-      -Wait
+  # Start the installer silently. This automatically starts the
+  # "$LOGGINGAGENT_SERVICE" service.
+  Log-Output 'Invoking Logging agent installer'
+  cd $tmp_dir
+  Expand-Archive $tmp_dir\td.zip
+  mv .\td\td-agent-bit-1.5.1-win64\ $LOGGINGAGENT_ROOT
+  cd $LOGGINGAGENT_ROOT
+
+  sc.exe create $LOGGINGAGENT_SERVICE binpath= "${LOGGINGAGENT_ROOT}\bin\fluent-bit.exe -c \fluent-bit\conf\fluent-bit.conf"
+  sc.exe failure $LOGGINGAGENT_SERVICE reset= 30 actions= restart/5000
```

Comment:
@jeremyje has a PR out to do this for Stackdriver; we should decide which values make sense to use here.

@jeremyje that PR might be obsolete now that we have this one.